### PR TITLE
Register mig.is-a.dev

### PR DESCRIPTION
**Pull request by miguelaram2016**
Register mig.is-a.dev with A record pointing to 75.2.60.5.

### CODE REVIEW

File: domains/mig.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "miguelaram2016",
+           "email": "",
+           "discord": "747499334383304755",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.sh5h8IcwANNjL4Cu_olrTjTuGNiBnihpvp8tYD3PQoJ0Qo8zJwh2wh93m0mxtB0JUmQyLHgMhqwNkRTfDkbQtIiMsJTMBpfwuekG-rohCXC3Bz1X674p58mTiI9BL1mYZjy4n3iNUltRrpEDIY_tQ91-deuB0vZmhFP7Ih-PU18_a7SFhrMTCGFnEpy5ioRBb_7-XaevqZU0ObJ9TXnuX5ZrcUs8dOKdPXQkUvh1T3X4ENkNkqxEX9SrC21ZUPsu0eBsinx8OWhVACBF-v6WsNErv4dYlxvlUF0s6i6H4ies_u47cXuD_ytOZVF19KRHv0mtC4Zq9FxKqvxQNpS2yA.ATDrqWU61Q7pAeI0iOYG3Q.WUBkDKZeYnntdHyh4t_qL56ZUvo4xiK6dF_DvM8i0aVxoooMhunELz8oFORqt_1lt3IMLw_aAWwzmBl4LtPa-frLObpHRe3QHeAhAb919XU.uvFZv2MiYxP-UoMM0DhLSA"
+        },
+    
+        "record": {
+            "CNAME": "mig.is-a.dev"
```
Comment (CuteDog5695) [inline domains/mig.json:10]: You can't point the domain to itself.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/mig.json (hunk 1 above)
Comment (miguelaram2016) [reply domains/mig.json:10]: Sorry, I am attempting to use Netlify and a bit confused about DNS,  but perhaps I got it now

**Comment by miguelaram2016** [comment]
Looks like I broke things :(

File: domains/mig.json (hunk 1 above)
Comment (CuteDog5695) [reply domains/mig.json]: @miguelaram2016 We literally have docs about Netlify [here](https://www.is-a.dev/docs/netlify/)

Comment (miguelaram2016) [reply domains/mig.json]: Going through the Docs now, though I mistakenly closed the PR

**Comment by miguelaram2016** [comment]
I hope I understood the docs correctly. I've updated the json file

**Review by iostpa — CHANGES_REQUESTED** [review]
Can you give a preview of your website?

**Comment by miguelaram2016** [comment]
> Can you give a preview of your website?

https://miguel-portfolio.netlify.app/

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**